.github/workflows/firebase.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy Firebase
+on:
+  push:
+    branches: [ 'feature/firebase' ]
```

Comment:
Ultimately this should changed, very likely to `branches: [ development, staging, master ]` To be discussed.